src/schemas/21-526EZ/schema.js

```diff
@@ -331,7 +309,34 @@ let schema = {
       required: ['servicePeriods', 'servedInCombatZone'],
       properties: {
         servicePeriods: {
-          $ref: '#/definitions/servicePeriods'
+          type: 'array',
+          items: {
+            type: 'object',
+            properties: {
+              serviceBranch: {
+                type: 'string',
+                enum: [
+                  'Air Force',
+                  'Air Force Reserves',
+                  'Army',
+                  'Army Reserves',
+                  'Coast Guard',
+                  'Coast Guard Reserves',
+                  'Marine Corp',
+                  'Marine Corps Reserves',
+                  'Navy',
+                  'Navy Reserves',
+                  'NOAA',
+                  'USPHS',
```

Comment:
Should we spell these acronyms out and provide the acronyms in parentheses instead?

Reply:
Good call. I'm going to double check with Alex.